src/java/com/google/cloud/bigquery/dwhassessment/hooks/logger/EventRecordConstructor.java

```diff
@@ -101,7 +103,7 @@ private GenericRecord getPreHookEvent(HookContext hookContext) {
         .set("RequestUser", getRequestUser(hookContext))
         .set("ExecutionMode", executionMode.name())
         .set("ExecutionEngine", conf.get("hive.execution.engine"))
-        .set("Queue", getQueueName(executionMode, conf))
+        .set("Queue", getSupposedQueueName(executionMode, conf))
```

Comment:
I think it's better to chose one of these 2 options - 

1) Keep the name as `getQueueName` and write a doc on top of the function describing that it's `supposed`
2) Change the `record field` name to reflect that it's `supposed`

Reply:
What does the word `supposed` mean in this context?

Reply:
If in post event we can't receive Queue name because of exception, we will have at least a queue, which was set by a user explicitly.

I was thinking about using a different column, but for some reason couldn't justify it – after all, it's the same queue name. WDYT?

Reply:
Sounds good!